htdocs/core/class/html.formfile.class.php

```diff
@@ -1300,7 +1300,7 @@ public function list_of_documents($filearray, $object, $modulepart, $param = '',
 					print '<!-- Line list_of_documents '.$key.' relativepath = '.$relativepath.' -->'."\n";
 					// Do we have entry into database ?
 					print '<!-- In database: position='.$filearray[$key]['position'].' -->'."\n";
-					print '<tr class="oddeven" id="row-'.($filearray[$key]['rowid'] > 0 ? $filearray[$key]['rowid'] : 'AFTER'.$lastrowid.'POS'.($i + 1)).'">';
+					print '<tr class="oddeven" id="row-'.(!empty($filearray[$key]['rowid'] && $filearray[$key]['rowid'] > 0) ? $filearray[$key]['rowid'] : 'AFTER'.$lastrowid.'POS'.($i + 1)).'">';
```

Comment:
syntax error here.

Reply:
Laurent
When I edit the file it's already corrected...

Reply:
Still some syntax errors. See travis report.

Reply:
I know that it should be 👍 (!empty($filearray[$key]['rowid']**)** && $filearray[$key]['rowid'] > 0)
but when I edit the file it's already done...